### PR TITLE
[DOC-2284]  Support replicating creating/dropping loading jobs (4.1.1)

### PR DESCRIPTION
**Pull request by zhihuapengtg**
[DOC-2284 ](https://graphsql.atlassian.net/browse/DOC-2284)  for ticket GLE-7089

### CODE REVIEW

**Comment by zhihuapengtg** [comment]
@arun-TG-PM @victorleeTG Please help review this PR:  Support replicating creating/dropping loading jobs in DR, thanks!

File: modules/cluster-and-ha-management/pages/crr-index.adoc [hunk 1]
```diff
@@ -27,16 +27,14 @@ The following information is automatically synced from the primary cluster to th
 * All schema change jobs
 * All users and roles
 * All queries in every graph. Queries that are installed in the primary cluster are automatically installed in the DR cluster.
+* All loading jobs
```
Comment (victorleeTG) [inline modules/cluster-and-ha-management/pages/crr-index.adoc]:
```suggestion
* All loading jobs [v4.1.1+]
```

File: modules/cluster-and-ha-management/pages/crr-index.adoc [hunk 2]
```diff
@@ -27,16 +27,14 @@ The following information is automatically synced from the primary cluster to th
...
 
 == Exclusions
 
 The following information and commands are *not* synced to the DR cluster:
 
 * GraphStudio metadata
 ** This includes graph layout data and user icons for GraphStudio.
```
Comment (victorleeTG) [inline modules/cluster-and-ha-management/pages/crr-index.adoc:37]:
```suggestion
** This includes graph layout data and user icons for GraphStudio.
* [v4.1.0 Preview] Loading job definitions are not synced, but the data is still loaded to the DR cluster.
```

**Comment by victorleeTG** [comment]
Replaced with a new PR targeted at 4.1.1-dev instead of 4.1.o.

**Pull request closed without merging**